### DISCUSSION TITLE
Issue with now Helper Returning Incorrect Year

### DISCUSSION THREAD

**Discussion by kamali-lama** [mockoon/mockoon#1635] [Help]
When using the `now 'MMDDYYYY'` helper to format the current date, it returns the year off one year.

Here is an example of the code I am using: `"reportDate": "{{now 'MMDDYYYY'}}"`

The expected output was 12/30/2024, but it is returning 12/30/2025.

**Comment by 255kb** [comment]
We are using date-fns for date formatting. You should use lower case "y" instead of uppercase. You will find an explanation [here](https://github.com/date-fns/date-fns/issues/1055).
I will update the uppercase occurences in the docs to avoid confusion.

**Reply by kamali-lama** [reply]
That fixes it. This discussion can be closed now.
